exchanges/request/request.go

```diff
@@ -200,6 +206,12 @@ func (r *Requester) doRequest(req *http.Request, p *Item) error {
 			}
 		}
 
+		if p.HeaderResponse != nil {
```

Comment:
This does not get set anywhere (it does in tests), so will always be nil. Should this be a check to see if the response header is nil?
Or is it that when you want to see the header response, you create a `p.HeaderResponse` in the request?

Reply:
Yeah so if you want to inspect it package side you will send a pointer to it via requester.Item{} So if you haven't set anything it will just continue running as normal.